e2e-tests/docs/CI.md

```diff
@@ -0,0 +1,135 @@
+# CI Steps and Processes for Testing
+
+This document provides a comprehensive overview of our Continuous Integration (CI) testing processes. It explains when and how different test jobs are triggered, where to monitor them, and details about the environments hosting these tests, along with guidance on how to access them.
+
+## GitHub Pull Requests
+
+When a new Pull Request (PR) is opened at [backstage-showcase](https://github.com/janus-idp/backstage-showcase), tests can be triggered in two ways:
+
+1. **Commenting `/ok-to-test`**: Only members of the janus-idp GitHub organization can mark the PR with this comment to validate it for testing.
+
+2. **Triggering Tests Post-Validation**: After a janus-idp member has validated the PR, anyone can trigger tests using the following commands:
```

Comment:
e2e-tests run as soon as a PR is opened. Users can re-trigger them for any reason, such as in case of failure, by using these commands. You can also add `/test e2e-tests` to the following commands.

Reply:
fixed, thank you!